src/main/js/src/setupProxy.js

```diff
@@ -337,13 +337,17 @@ module.exports = function (app) {
   });
 
   app.post('/yki/api/virkailija/organizer', (req, res) => {
-    try {
-      organizers.push(req.body);
-      res.send({ success: true });
-    } catch (err) {
-      printError(req, err);
-      res.status(404).send(err.message);
-    }
+    const mockCall = () => {
+      try {
+        organizers.push(req.body);
+        res.send({ success: true });
+      } catch (err) {
+        printError(req, err);
+        res.status(404).send(err.message);
+      }
+    };
+
+    useLocalProxy ? proxyPostCall(req, res) : mockCall();
```

Comment:
Changed this logic to actually store organizer data if proxy backend is used. Nothing to do with the actual task.